src/inspector/models/notation/tempos/temposettingsmodel.cpp

```diff
@@ -27,18 +27,23 @@
 
 using namespace mu::inspector;
 
-TempoSettingsModel::TempoSettingsModel(QObject* parent, IElementRepositoryService* repository)
+TempoSettingsModel::TempoSettingsModel(QObject* parent, IElementRepositoryService* repository, InspectorModelType modelType)
     : AbstractInspectorModel(parent, repository)
 {
-    setModelType(InspectorModelType::TYPE_TEMPO);
-    setTitle(qtrc("inspector", "Tempo"));
+    Q_ASSERT(modelType == InspectorModelType::TYPE_TEMPO || modelType == InspectorModelType::TYPE_TEMPO_RESTORE_PREVIOUS);
+    setModelType(modelType);
+    if (modelType == InspectorModelType::TYPE_TEMPO) {
+        setTitle(qtrc("inspector", "Tempo"));
+    } else {
+        setTitle(qtrc("inspector", "Restore previous tempo (a tempo)"));
+    }
     setIcon(ui::IconCode::Code::METRONOME);
```

Comment:
For readability, it would be preferable to add a few newlines:
```suggestion
    Q_ASSERT(modelType == InspectorModelType::TYPE_TEMPO || modelType == InspectorModelType::TYPE_TEMPO_RESTORE_PREVIOUS);
    
    setModelType(modelType);
    
    if (modelType == InspectorModelType::TYPE_TEMPO) {
        setTitle(qtrc("inspector", "Tempo"));
    } else {
        setTitle(qtrc("inspector", "Restore previous tempo (a tempo)"));
    }
    
    setIcon(ui::IconCode::Code::METRONOME);
```